design.md

```diff
@@ -1,13 +1,53 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+Our target is the general public.  No programming knowledge would be required; thus, it is 
+a syntax readable by anyone who wants to play with Picobot.
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+We wanted to make the syntax more readable than the original syntax.  We did this by
+exchanging numbers for strings when describing state. We also modularized the rules more
+clearly by creating separate objects for each state.
+
+Furthermore, we wanted to simplify the program by limiting four possible states
+corresponding to the possible directions the Picobot can move in. By doing this,
+it is more clear to reader and user when we would change direction and states.
+As opposed to having states anywhere from 1 - 99, and having no easy understanding
+what a given numerical state represents in terms of directional movements, we 
+can more clearly organize what should happen at any given move when reading the
+code. 
+
 
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+Because we do away with arbitrary numerical state and replace it with four 
```

Comment:
That's a good point. As you mentioned below, our design definitely limits what the user can do. We were only taking into account solving the empty room map and the maze map, and did not even think about the possibility of needing >4 states.